configs/templates/nco/approved_list.txt

```diff
@@ -0,0 +1,225 @@
+awscli
```

Comment:
Would it make sense to sort this alphabetically?

Reply:
No, it's based on a spreadsheet from NCO and I'm just doing some grep magic on the concretization output, so it's fine as it is.